packages/ckeditor5-table/theme/tablecolumnresize.css

```diff
@@ -12,17 +12,20 @@
 	--ck-table-column-resizer-position-offset: calc(var(--ck-table-column-resizer-width) * -0.5 - 0.5px);
 }
 
+.ck-content .table .ck-table-resized {
+	table-layout: fixed;
+}
+
 .ck-content .table table {
 	overflow: hidden;
```

Comment:
This is doable e.g. by setting the inline styles separately for each resizer - but it adds a lot of complexity. We'd have to calculate the height of the table, the height of the cell and the distance from the table beginning to the resizer's cell beginning - then we could set the `top` and `bottom` styles to cover exactly the table height. But that's quite a bit of work and it would have to be recalculated whenever something changes in the table layout.

I think we should extract this issue and I'll put the note above as a solution proposal there.

Reply:
Yea, that's in line with what I've been thinking about it. Adding manual calculation is definitely too much work for now.

Reply:
Extracted to #11973